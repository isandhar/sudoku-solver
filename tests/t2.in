D*61**9**2**5**0
C****1********F*
B****0*2D***34*8
89E*****4*0**CA*
*F*0***D15*82B93
*69*7*0B3A****8*
*2*D1**E***B**C5
A***C*2*E04*F***
***F*B182*******
*4*5A**7*CD08**F
**8****C**7***42
9**E25***F3**7D*
*D23******EA0***
48A***E***5FB*36
*E57B*F*********
************7*2*
