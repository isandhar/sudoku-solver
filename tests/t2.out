D7613894F2AC5EB0
C034D1A658BE92F7
B5FAE0C2D1973468
89E257BF4306DCA1
EF704A6D15C82B93
569C7F0B3A2DE184
324D198E76FBA0C5
A1B8C325E049F67D
73DF9B182E64C50A
2415A637BCD089EF
0B86FEDCA9751342
9ACE25408F3167DB
1D2384796BEA0F5C
48A902E1C75FBD36
6E57BCF30D824A19
FC0B6D5A9413782E
